blocks/primlancers/primlancers.css

```diff
@@ -0,0 +1,208 @@
+.primlancers {
+    padding: 80px 0 60px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.primlancers__title {
```

Comment:
There is a heading block to be used here. Us that instead and remove the margin from the overlay class